modules/account-lib/test/unit/coin/avaxc/transactionBuilder/walletInitializationBuilder.ts

```diff
@@ -128,5 +129,40 @@ describe('AvaxC Wallet Initialization Builder', function () {
 
       txBuilder.build().should.be.rejectedWith('Invalid transaction: wrong number of owners -- required: 3, found: 2');
     });
+
+    it('Should getting same tx from raw tx', async function () {
+      initTxBuilder();
+      txBuilder.owner(testData.OWNER_1.ethAddress);
+      txBuilder.owner(testData.OWNER_2.ethAddress);
+      txBuilder.owner(testData.OWNER_3.ethAddress);
+      txBuilder.sign({ key: testData.OWNER_1.ethKey });
+      const tx = await txBuilder.build();
+      const txBuiderFromRaw = getBuilder('tavaxc') as TransactionBuilder;
+      txBuiderFromRaw.from(tx.toBroadcastFormat());
+      const txFromRaw = await txBuiderFromRaw.build();
+      should.deepEqual(tx.toJson(), txFromRaw.toJson());
+      should.deepEqual(tx.toBroadcastFormat(), txFromRaw.toBroadcastFormat());
+      should.deepEqual(tx.id, txFromRaw.id);
+    });
+
+    it('Should getting same tx from raw tx ETH', async function () {
```

Comment:
@CapnMigraine this test is copied in modules/account-lib/test/unit/coin/eth/transactionBuilder/walletInitialization.ts
Should I erase from here?